.env.example

```
@@ -10,4 +10,5 @@ REDIS_USERNAME=
 MXBAI_API_KEY=
 RESEND_API_KEY=
 RESEND_NOREPLY=
-CLIENT_MODE=ONLINE #set LOCAL to run local Ollama instead of OpenAI and Groq API
\ No newline at end of file
+CLIENT_MODE=ONLINE #set LOCAL to run local Ollama instead of OpenAI and Groq API
+OLLAMA_ENDPOINT= # !mandatory! set the ollama endpoint
```

```suggestion
OLLAMA_ENDPOINT=
```

We are adding a fallback in the code